rest/utilities_testing.go

```diff
@@ -500,7 +504,12 @@ func (rt *RestTester) GetDatabase() *db.DatabaseContext {
 
 // CreateUser creates a user with the default password and channels scoped to a single test collection.
 func (rt *RestTester) CreateUser(username string, channels []string) {
-	response := rt.SendAdminRequest(http.MethodPut, "/{{.db}}/_user/"+username, GetUserPayload(rt.TB(), "", RestTesterDefaultUserPassword, "", rt.GetSingleTestDatabaseCollection(), channels, nil))
+	var response *TestResponse
+	if rt.AdminInterfaceAuthentication {
+		response = rt.SendAdminRequestWithAuth(http.MethodPut, "/{{.db}}/_user/"+username, GetUserPayload(rt.TB(), "", RestTesterDefaultUserPassword, "", rt.GetSingleTestDatabaseCollection(), channels, nil), base.TestClusterUsername(), base.TestClusterPassword())
```

Comment:
I think this code is fine but I've hit this type of bug before. I wonder (for future) if we should do something so that we always call `SendAdminRequestWithAuth` and then it returns empty string if the admin interface is configured without authentication.

Reply:
Yeah I wasn't quite sure how far to push that change down. I can see legitimate uses for sending unauthorized requests to ensure the negative cases are handled, but that does seem like a more special case (e.g. `SendAdminRequestNoAuth`)